### DISCUSSION TITLE
Card Observations

### DISCUSSION THREAD

**Discussion by nullableVoidPtr** [nullableVoidPtr/nfcf-info#4] [Analysis]
This thread is for recording NFC-F tags.

_Please follow the below format!_

```md
**Description:**

**Picture (optional):**

**Manufacturer code:** `0000`

**IC Type:** `00`

**ROM Type:** `00`
```

The Manufacturer Code is the first two bytes of the Manufacturer ID (IDm).

The IC Type and ROM Type are from the  2nd and 1st byte of the Manufacturer Parameter (PMm).

Both the IDm and PMm can be retrieved with a polling command.

**Comment by hanpuliu-charles** [comment]
**Description:**
Suica 2 in 1 (Iguca)
**Picture (optional):**
![image](https://github.com/user-attachments/assets/40a72b38-f84f-4536-b783-74816c8985e2)

**Manufacturer code:** `1201`

**IC Type:** `31`

**ROM Type:** `06`

**Description:**
Early Suica with Penguin and iO card marking
**Picture (optional):**
![image](https://github.com/user-attachments/assets/66c75d8a-0eba-4981-aea9-683186540c67)

**Manufacturer code:** `0101`

**IC Type:** `01`

**ROM Type:** `04`

More cards to come in early Jan, I will share more about octopus cards, including ones that don't seem to be made by Sony(with dual type A/F interface). Thanks for the hardwork consolidating all the information!
